kendzi3d-jogl-model-render/src/main/java/kendzi/jogl/model/render/ModelRender.java

```diff
@@ -74,18 +78,24 @@ public void render(GL2 gl, Model model) {
         // gl.glCullFace(GL.GL_FRONT);
         // }
 
-        if (model.useLight) {
+        if (model.useLight)
             gl.glEnable(GLLightingFunc.GL_LIGHTING);
-        }
-        if (model.useTexture) {
-            gl.glEnable(GL.GL_TEXTURE_2D);
-        } else {
-            gl.glDisable(GL.GL_TEXTURE_2D);
-        }
 
         draw(gl, model);
 
+        if (model.useLight)
```

Comment:
Ok, one project uses them, another does not.
I do not really care, either way is fine.

If it annoys you having to tell people your preference for the project,
maybe you can put some pref files for eclipse IDE to checkout / clone.

I.e. if you pull (or checkout) josm, a project specific pref file for eclipse is included that defines  for the editor  which save actions to take.  Eclipse / the editor then uniformly handles this preference for all devs (and per project).

E.g. you could push .settings/org.eclipse.jdt.ui.prefs with lines like
sp_cleanup.always_use_blocks=true
sp_cleanup.never_use_blocks=false
sp_cleanup.use_blocks=false
sp_cleanup.use_blocks_only_for_return_and_throw=true

Josm [pushes org.eclipse.jdt.ui.prefs](https://josm.openstreetmap.de/browser/josm/trunk/.settings/org.eclipse.jdt.ui.prefs) with
sp_cleanup.always_use_blocks=false
sp_cleanup.never_use_blocks=false
sp_cleanup.use_blocks=false
sp_cleanup.use_blocks_only_for_return_and_throw=true

Reply:
I see you have .settings/  in .gitignore

If you were to include a prefs file like suggested above, you'd also have to update .gitignore to allow it being tracked by git.

Reply:
I simply hate if's without block. Settings are ignored in git because not everybody uses eclipse for coding. In past had some description how to setup eclipse for coding, but cant find it now, it was lost in time.